cmd.js

```diff
@@ -110,5 +112,38 @@ module.exports = Command = {
       return rika.notHere(name)
     }
     return rika.dontKnow(name)
+  },
+  getUserFloor: async function (_name) {
+    const devices = await User.find({ name: _name })
+    const activeDevice = await getActiveDevice(devices)
+    const name = capitalize(_name)
+    if (activeDevice) {
+      const currentFloor = activeDevice.floor
+      return currentFloor
+    }
+    return null
+  },
+  getOfficeMap: async function () {
+    const names = await User.distinct('name')
+    const users = []
+    await Promise.all(
```

Comment:
Since `map` already return an array, can we assign `users` like....?
```js
const users = await Promise.all(...)
```

Reply:
👍

Reply:
Okay no need to change I forgot the data in `users` should exclude `undefined` or `null`.